go.mod

```diff
@@ -266,6 +261,7 @@ require (
 replace (
 	github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-20180902122335-24332e2d58ab
 	github.com/gocql/gocql => github.com/grafana/gocql v0.0.0-20200605141915-ba5dc39ece85
+	github.com/prometheus/common => github.com/prometheus/common v0.30.0
```

Comment:
a lot of these replaces exist due to our cortex dependency, can any of them be removed with this upgrade?

Reply:
Sure, I'll figure out what can be removed.